scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -198,7 +198,7 @@ abstract class LocalScanner(
     }
 
     override suspend fun scanPackages(
-        packages: List<Package>,
+        packages: Collection<Package>,
```

Comment:
I don't quite understand why this step makes it easier to switch to sets later on?

Reply:
Because you can pass (the implicitly created) `Set<Package>` instead of `List<Package>`.

Reply:
Also see the commit message of https://github.com/oss-review-toolkit/ort/pull/1344/commits/bd946bb92c8ea2d7e2c06689e5fd42d77800f459.